test/binaryen.js/kitchen-sink.js

```diff
@@ -676,13 +676,14 @@ function test_core() {
   assert(table.base === "");
   assert(table.initial === 0);
   assert(table.max === 2);
-
+  
   module.removeTable("t1");
   assert(module.getNumTables() === 0);
-
-  // Legacy
-  module.setFunctionTable(1, 0xffffffff, [ binaryen.getFunctionInfo(sinker).name ]);
+  
+  module.addTable("0", 1, 0xffffffff);
+  module.addActiveElementSegment("0", "0", [ binaryen.getFunctionInfo(sinker).name ]);
```

Comment:
It would be good to use different names for the table and segment so they can be verified to end up in the right place in the test output.